src/test/java/org/broadinstitute/dsde/consent/ontology/datause/builder/ConsentRestrictionBuilderTest.java

```diff
@@ -219,4 +221,24 @@ public void testCSdulUC3() {
         Assert.assertEquals(restriction, ConsentUseCases.CSdulUC3);
     }
 
+    @Test
+    public void testCommercialUse() {
+        DataUse dataUse = new DataUseBuilder().
+                setCommercialUse(true).
+                build();
+        UseRestriction restriction = restrictionBuilder.buildUseRestriction(dataUse);
+        Assert.assertTrue(restriction.toString().contains(NON_PROFIT));
+        Assert.assertTrue(restriction.toString().contains("\"type\":\"not\""));
+    }
+
+    @Test
+    public void testCommercialUseExcluded() {
```

Comment:
Should there be a test for `getCommercialUse` returning `null`, since the main code tries to handle it?

Reply:
Nice catch, and yes, there should